Cargo.lock

```diff
@@ -363,6 +372,15 @@ version = "0.1.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "597bb81c80a54b6a4381b23faba8d7774b144c94cbd1d6fe3f1329bd776554ab"
 
+[[package]]
+name = "bincode"
```

Comment:
why is this making a comback?

Reply:
what? How did this get here?

Reply:
Ok, it was via quic-rpc, where I forgot to remove the feature from tokio-serde.

It is gone now:
```
❯ cargo tree --all-features -i bincode
error: package ID specification `bincode` did not match any packages
```

Reply:
But the `Cargo.lock` file here still has `bincode` here, right?
Why does `cargo tree` tell you it's not in there, but the `Cargo.lock` file in this PR still adds it?

Reply:
@rklaehn did you give up fixing this?

Reply:
will be in the next PR.